ocs_ci/ocs/constants.py

```diff
@@ -3462,3 +3464,43 @@
 MCLOCK_HIGH_CLIENT_OPS = "high_client_ops"
 MCLOCK_BALANCED = "balanced"
 MCLOCK_HIGH_RECOVERY_OPS = "high_recovery_ops"
+
+
+def get_noobaa_cli_path():
+    """
+    Get the appropriate NooBaa CLI path based on OCS version.
+
+    For OCS version >= 4.20: Returns path to odf-cli
+    For OCS version < 4.20: Returns path to mcg-cli
+
+    Returns:
+        str: Path to the CLI binary to use for NooBaa operations
+    """
+    from ocs_ci.utility import version
+
+    ocs_version = version.get_semantic_ocs_version_from_config()
+
+    if ocs_version >= version.VERSION_4_20:
+        return ODF_CLI_LOCAL_PATH
+    else:
+        return NOOBAA_OPERATOR_LOCAL_CLI_PATH
+
+
+def get_noobaa_cli_command_prefix():
+    """
+    Get the appropriate command prefix for NooBaa CLI operations.
+
+    For OCS version >= 4.20: Returns 'noobaa' (for odf-cli noobaa <command>)
+    For OCS version < 4.20: Returns '' (for mcg-cli <command>)
+
+    Returns:
+        str: Command prefix to insert between CLI binary and actual command
+    """
+    from ocs_ci.utility import version
+
+    ocs_version = version.get_semantic_ocs_version_from_config()
+
+    if ocs_version >= version.VERSION_4_20:
+        return "noobaa"
+    else:
+        return ""
```

Comment:
Since both method returns `str`, I would request you to combine and return path along with prefix.

I would also request you to move this code to utils.py file since this file mostly speaks about constants